lib/tasks/launch-browserstack-browser.js

```diff
@@ -1,34 +1,10 @@
 #!/usr/bin/env node
 const buildNameFromEnv = require('../utils/build-name-from-env');
 
-if (!process.env.BROWSERSTACK_USERNAME || !process.env.BROWSERSTACK_ACCESS_KEY) {
-  throw new Error("Please set the 'BROWSERSTACK_USERNAME' and 'BROWSERSTACK_ACCESS_KEY' environment variables");
-}
-
-let workerId = 0;
-let BrowserStack = require('browserstack');
-
-let client = BrowserStack.createClient({
-  username: process.env.BROWSERSTACK_USERNAME,
-  password: process.env.BROWSERSTACK_ACCESS_KEY
-});
-
-['SIGINT', 'SIGTERM', 'SIGHUP'].forEach(function(signal) {
-  process.on(signal, function() {
-    console.log("Closed BrowserStack Worker process " + signal);
-    if (client !== null) {
-      client.terminateWorker(workerId, function() {
-        process.exit();
-      });
-    } else {
-      process.exit();
-    }
-  });
-});
-
 let build = buildNameFromEnv();
```

Comment:
I think it is fine to move the yargs down to get help working without setting the environment variables, though I don't know how likely it is that `npx ember-cli-browserstack --help` would ever be used, since that runs this script -- that launches the browser, not the addon and it's commands. This script launches the browsers and should only be used by testem, not on the CLI. If wanting to launch BrowserStack browsers from the CLI, their tools should be used: https://www.browserstack.com/local-testing#command-line

`buildNameFromEnv()` uses the environment variables and since it will be a default in the args,  if we keep the movement of the assertions below the args, it should be updated to output something reasonable when the Env vars are missing.

Reply:
I added default value like "name" property has with crypto hex.